src/sharedHooks/useStorage.ts

```diff
@@ -0,0 +1,12 @@
+import { useMMKVBoolean } from "react-native-mmkv";
```

Comment:
I noticed you've been saving TypeScript files more often -- could you add this to the next mobile meeting and tell us a little about when and why you're using it? I suspect we all want to be moving in the direction of TypeScript, rather than Flow, and I think it would be a good thing for us to collaborate on. I added TypeScript extensions to Tailwind, so afaik we should be able to use TypeScript in all of our UI components now.

Reply:
Can do. But there is not much to say I guess. Whenever I create a new file that has no dependency on any of our own Flow-typed JS files I use Typescript.

Reply:
Thanks. I think it would be helpful to understand why you're using TS, not Flow, even if that reason is just "this is the direction the industry is going."